packages/wrangler/e2e/miniflare-mixed-mode-resources.test.ts

```diff
@@ -0,0 +1,355 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/consistent-type-imports */
```

Comment:
why? why not just auto-fixing the imports? 🤔 

(I'm getting the feeling that maybe you committed these two `eslint-disable` rules by mistake 🤔)

Reply:
Oops, yeah, the consistent-type-imports rule can be removed, I think